portfolio/src/main/webapp/statistics.html

```diff
@@ -166,11 +166,55 @@ <h4>Leave a marker on the map for protests you will be attending!</h4>
             </button>
           </div>
         </form>
+        <hr />
+        <h6>Find Cases Of Police Brutality In Your Area</h6>
+        <div class="row align-items-stretch mb-5">
+          <div class="col-md-auto">
+            <input
+              class="form-control"
+              id="address"
+              name="address"
+              type="text"
+              placeholder="Your Address *"
+              required
+              data-validation-required-message="Please Enter Your Address."
+            />
+          </div>
+          <input id="state" type="text" size="5" placeholder="State *" />
+          <p class="help-block text-danger"></p>
+          <div class="text-center">
+            <label for="radius">
+              Select a Radius In Miles That You Would Like To See Pins Within:
+            </label>
+            <select name="radius" id="radius">
+              <option value="5">5</option>
+              <option value="10">10</option>
+              <option value="15">15</option>
+              <option value="50">50</option>
+              <option value="100">100</option>
+            </select>
```

Comment:
just a suggestion, depends on personal preference: instead of saying "Select a Radius in Miles", add miles to the options instead, so it'd be "5 miles", "10 miles", etc.